docs/examples/carousel/index.html

```diff
@@ -129,7 +129,7 @@ <h2 class="featurette-heading">First featurette heading. <span class="text-muted
           <p class="lead">Donec ullamcorper nulla non metus auctor fringilla. Vestibulum id ligula porta felis euismod semper. Praesent commodo cursus magna, vel scelerisque nisl consectetur. Fusce dapibus, tellus ac cursus commodo.</p>
         </div>
         <div class="col-md-5">
-          <img class="featurette-image img-fluid center-block" data-src="holder.js/500x500/auto" alt="Generic placeholder image">
+          <img class="featurette-image img-fluid m-x-auto" data-src="holder.js/500x500/auto" alt="Generic placeholder image">
```

Comment:
Correct, these rendered as expected w/o need for additional `.d-block` (same for line 90 in `docs/index.html`). Only the `docs/images.md` needed it to match previous styling.